test/integration/connect/envoy/run-tests.windows.sh

```diff
@@ -77,6 +77,7 @@ function init_workdir {
   # don't wipe logs between runs as they are already split and we need them to
   # upload as artifacts later.
   rm -rf workdir/${CLUSTER}
+  rm -rf workdir/logs
   mkdir -p workdir/${CLUSTER}/{consul,consul-server,register,envoy,bats,statsd,data}
```

Comment:
Check lines 76 to 78. On Linux they deliberately don't wipe the logs, they use them for some other purpose.